templates/supervisord.conf.j2

```diff
@@ -30,10 +30,10 @@ password = {SHA}{{ supervisor_password|hash('sha1') }}
 {% endif %}
 
 [supervisorctl]
-serverurl = http://localhost:{{ supervisor_inet_http_server_port }}
+serverurl = http://{{ 'localhost' if supervisor_inet_http_server_port.split(':')[0] == '*' or supervisor_inet_http_server_port.split(':')[0] == '' else supervisor_inet_http_server_port.split(':')[0] }}:{{ supervisor_inet_http_server_port.split(':')[1] }}
 {% if supervisor_inet_http_server_password_protect %}
 username = {{ supervisor_user }}
-password = {SHA}{{ supervisor_password|hash('sha1') }}
+password = {{ supervisor_password }}
```

Comment:
This seems like it could be a breaking change, depending on how people are using supervisor — from the docs: http://supervisord.org/configuration.html#inet-http-server-section-values

> The password required for authentication to this HTTP server. This can be a cleartext password, or can be specified as a SHA-1 hash if prefixed by the string {SHA}. For example, {SHA}82ab876d1387bfafe46cc1c8a2ef074eae50cb1d is the SHA-stored version of the password “thepassword”.
> 
> Note that hashed password must be in hex format.

Could you break out this change / discussion into a separate PR so it can be evaluated separately?

Reply:
Sorry for any inconvenience. Just a big fan and couldn't resist mentioning your latest work.

Reply:
@geerlingguy The doc you mention is about the inet_http_server section. The change is in the supervisorctl section.
http://supervisord.org/configuration.html#supervisorctl-section-values

> The password to pass to the supervisord server for use in authentication. This should be the cleartext version of password from the supervisord server configuration for the port or UNIX domain socket you’re attempting to access. This value cannot be passed as a SHA hash. Unlike other passwords specified in this file, it must be provided in cleartext.

Reply:
Ah...

> This value cannot be passed as a SHA hash. Unlike other passwords specified in this file, it must be provided in cleartext.

Okay :)